subprojects/testing-jvm/src/integTest/groovy/org/gradle/testing/junit/JUnitCategoriesIntegrationSpec.groovy

```diff
@@ -122,6 +122,7 @@ public class MyTest {
         |testing {
         |   suites {
         |       test {
+        |           useJUnit()
```

Comment:
The changes to this test are to produce a much cleaner test buildScript, the previous one, captured from a breakpoint for each run in this test, had some misleading and irrelevant duplicated content.